SPEC.md

```diff
@@ -163,6 +159,10 @@ in that state. They return `false` otherwise:
 - `#crashed?`
 - `#recycling?`
 
+- `#log`
```

Comment:
Commonly used term is `logger`, which IMO reads better. The `log` is kinda confusing since it might be both, a verb and a noun, and neither is aptly describing what it actually is, i.e. a logger instance. I know it follows the class name, but this IMO is a poor naming choice (which also breaks established language naming conventions).

Reply:
That's the _one_ thing that remained the same :joy: 

Hum, I'm debating this. I think `log.error` reads nicely as plain English, in comparison to `logger.error` for example. Same for `agent.log.error` as can be found below in the patch.

Reply:
Yeah, it makes sense on one level (plain English), then it doesn't on another one (code-wise) 😅